.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2017
+  },
+  "plugins": [
+    "prettier"
+  ],
+  "rules": {
+    "prettier/prettier": ["error", {
```

Comment:
JSON checking of this file please
and if it supports yaml config, that would be preferred

Reply:
Yes eslint does support yaml config